### ISSUE TITLE
OnlineSigningActivity.getEnvelopeViewModel UninitializedPropertyAccessException

### ISSUE THREAD

**Issue by JKuliska-pcty** [docusign/mobile-android-sdk#53]
Hello,
we're using DocuSign Android SDK in our app for launching the document signing flow and there are some reports from Firebase that it is occassionaly crashing.

### Version used
1.9.5
### Reproducability
I wasn't able to reproduce this issue even with _**Don't keep activities**_ and putting the app into background

### The way we launch DocuSign is:
```kotlin
DocuSign
    .getInstance()
    .getSigningDelegate()
    .launchCaptiveSigning(
        context = context,
        envelopeId = envelopeId,
        signingURL = url,
        listener = {},
        recipientId = null,
)
```

### Stacktrace

```
Fatal Exception: kotlin.UninitializedPropertyAccessException: lateinit property envelopeViewModel has not been initialized
       at com.docusign.androidsdk.ui.activities.OnlineSigningActivity.getEnvelopeViewModel(OnlineSigningActivity.kt:98)
       at com.docusign.androidsdk.ui.activities.OnlineSigningActivity.restartSigning(OnlineSigningActivity.kt:711)
       at com.docusign.androidsdk.ui.fragments.OnlineSigningFragment.genericConfirmationPositiveAction(OnlineSigningFragment.kt:1303)
       at com.docusign.androidsdk.ui.activities.OnlineSigningActivity.genericConfirmationPositiveAction(OnlineSigningActivity.kt:1058)
       at com.docusign.androidsdk.ui.fragments.GenericConfirmationDialogFragment.onCreateDialog$lambda$8$lambda$3$lambda$2(GenericConfirmationDialogFragment.kt:81)
       at com.docusign.androidsdk.ui.fragments.GenericConfirmationDialogFragment.$r8$lambda$bQ2kyosWgg6Kl8R5RbHRE-9xKbo()
       at com.docusign.androidsdk.ui.fragments.GenericConfirmationDialogFragment$$ExternalSyntheticLambda0.onClick(D8$$SyntheticClass)
       at androidx.appcompat.app.AlertController$ButtonHandler.handleMessage(AlertController.java:167)
       at android.os.Handler.dispatchMessage(Handler.java:106)
       at android.os.Looper.loopOnce(Looper.java:226)
       at android.os.Looper.loop(Looper.java:313)
       at android.app.ActivityThread.main(ActivityThread.java:8762)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:604)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1067)
```

**Comment by naveentds** [comment]
Hi
Can you let us know the crash count for this issue from your Firebase project.
We can also try to reproduce this issue on our end.

**Comment by JKuliska-pcty** [comment]
Thank you for looking into it.

We've seen 23 crashes in the last 2 weeks (since the DocuSign integration was released). 
The devices differ very much (basically each crash a different device)

**Comment by naveentds** [comment]
Hi
We released SDK v1.9.6. Please try it out and let us know how it goes.

**Comment by naveentds** [comment]
We hope the issue is resolved. In case if there are any issues, please let us know. 
Closing this issue for now.
